ajax/libs/jQRangeSlider/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jQRangeSlider",
+  "filename": "jQRangeSlider.min.js",
+  "version": "5.7.1",
+  "homepage": "http://ghusse.github.com/jQRangeSlider/",
+  "authors": {
+    "name": "Guillaume Gautreau",
+    "url": "http://www.ghusse.com",
+    "email": "guillaume+jqrangeslider@ghusse.com"
+  },
+  "description": "A javascript slider selector that supports dates",
+  "keywords": [
+    "date",
+    "slider",
+    "range",
+    "value",
+    "selection",
+    "select",
+    "ui",
+    "widget",
+    "number",
+    "scroll"
+  ],
+  "license": "GPL",
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/ghusse/jQRangeSlider"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ghusse/jQRangeSlider.git",
+    "basePath": "",
+    "files": [
+      "jQ*.js",
+      "css/**/*"
```

Comment:
We don't need this:
https://github.com/ghusse/jQRangeSlider/blob/v4.2.9/css/icons/.gitignore